components/intake.py

```diff
@@ -91,6 +92,8 @@ def get_cube_distance(self) -> float:
     @magicbot.feedback
     def is_cube_contained(self) -> bool:
         """Check whether a cube is in the containment mechanism."""
+        if hal.isSimulation():
+            return True
```

Comment:
Wouldn't actually setting the things in the physics be a better way of doing this? Especially if you're just testing with a switch…